open-grant-proposals/ars-go-filecoin-api.md

```diff
@@ -0,0 +1,110 @@
+# Open Grant Proposal: `go-filecoin-api-client`
+
+**Name of Project:** `go-filecoin-api-client`
+
+**Proposal Category:** `devtools-libraries`
+
+**Proposer:** 	[@waynewyang](https://github.com/waynewyang)
+
+**Do you agree to open source all work you do on behalf of this RFP and dual-license under MIT and APACHE2 licenses?** 	"Yes"
+
+# Project Description
+
+Currently, filecoin have not provided the corresponding filecoin-api of go version. Every developer must implement their own set of code logic to access filecoin when using go-lang.
```

Comment:
Hi team! Both lotus and go-filecoin already expose an API that can be used from go programs.